tests/unit/test_api_handlers.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/python3
+
+# sparql_adapter.py
+# Date:  27/12/2020
+# Author: Laurentiu Mandru
+# Email: mclaurentiu79@gmail.com
+
+from lam4doc.entrypoints.api.handlers import upload_rdfs
+
+
+def test_upload_rdfs_dataset_mandatory():
+    result = upload_rdfs({}, None, None, None)
+    assert result[0] == 'Dataset name is required'
+    assert result[1] == 400
+
+
+def test_upload_rdfs_at_least_one_file_mandatory():
```

Comment:
The current tests are execellent, and a few more would be nice :)

Reply:
We can talk about this. There's not much unit testing you can do since we use an adapter.